library/src/test/java/com/a99/rxplaces/RxAutocompleteStressTest.kt

```diff
@@ -43,17 +42,16 @@ class RxAutocompleteStressTest {
 
     // then
     testSubscriber.assertNoErrors()
-    testSubscriber.assertValueCount(3)
+    testSubscriber.assertValueCount(1)
 
-    verify(repository, times(3)).query(any(), any())
+    verify(repository, times(1)).query(any(), any())
```

Comment:
If the `times()` are `1`, you do not need to pass it as parameter. Please change it to `verify(repository).query()`